Dockerfile_schedule

```diff
@@ -0,0 +1,35 @@
+# https://hub.docker.com/_/debian
```

Comment:
all schedule requests should go via the runner now, which should mean there is no need to manually run the scheduler.

Reply:
Gotcha - good to know.
Dropping